src/globus_cli/commands/streams/tunnel/update.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+import uuid
+
+import click
+import globus_sdk
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import OMITTABLE_STRING, command
+from globus_cli.termio import display
+
+from ._common import TUNNEL_STANDARD_FIELDS
+
+
+@command("create", short_help="Create a Globus Tunnel.")
```

Comment:
This is the `update` command.

```suggestion
@command("update", short_help="Update a Globus tunnel.")
```